codegen/stub_api.py

```diff
@@ -227,7 +234,7 @@ def _process_variable(node: ElementTree.Element, indent: str = "") -> str:
         """
         var_type = node.get("type", "Any")
         if var_type.startswith("ENS_"):
-            var_type = f"Type[{var_type}]"
+            var_type = f"'{var_type}'"
```

Comment:
The autocompletion doesn't like "Type". So I am using the '' syntax, which fixes the autocompletion for all the cores objects